### ISSUE TITLE
string with bslash

### ISSUE THREAD

**Issue by sloanyyc** [dpranke/pyjson5#23]
son5.loads('{a:"a/"}')
ValueError: :1 Unexpected "/" at column 7
such as 'http://domain/' can't jsonfiy

**temp fix**
`# comment /usr/local/lib/python3.7/site-packages/json5/parser.py

def _sqchar__c2_(self):
        self._push('sqchar__c2')
        self._seq([# lambda: self._not(self._bslash_),
                   lambda: self._not(self._squote_),

def _dqchar__c2_(self):
self._push('dqchar__c2')
self._seq([# lambda: self._not(self._bslash_), # this line
lambda: self._not(self._dquote_),`

**Comment by dpranke** [comment]
Hi,

Thanks for the bug report. However, I don't think I understand what you're reporting.

You said "blash" in the subject, but your example is using forward slashes.

Also, if I test the example you give with the latest version of 

```
$ python3
>>> import json5
>>> json5.VERSION
'0.6.1'
>>> print(json5.loads(r'{a:"a/"}'))
{'a': 'a/'}
>>> print(json5.loads(r'{a:"a\"}'))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/src/pyjson5/json5/lib.py", line 59, in loads
    raise ValueError(err)
ValueError: <string>:1 Unexpected end of input at column 9
>>> print(json5.loads(r'{a:"a\\"}'))
{'a': 'a\\'}
>>> ^D
```

it all seems to work like I'd expect. Are you using the latest version? Maybe I'm missing something?

**Comment by sloanyyc** [comment]
json5.loads('{a:"a**\\/**"}')
ValueError: :1 Unexpected "/" at column 7
such as 'http://domain/' can't jsonfiy

Sorry, bug git edit translate \\/ as /

**Comment by sloanyyc** [comment]
In [5]: import json5                                                            

In [6]: json5.VERSION                                                           
Out[6]: '0.6.1'

In [7]: print(json5.loads(r'{a:"a\\/"}'))                                        
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-7-30331cfc6f55> in <module>
----> 1 print(json5.loads(r'{a:"a\\/"}'))

~/venv/lib/python3.7/site-packages/json5/lib.py in loads(s, encoding, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook)
     57     ast, err, newpos = parser.parse()
     58     if err:
---> 59         raise ValueError(err)
     60 
     61     def _fp_constant_parser(s):

ValueError: <string>:1 Unexpected "/" at column 7

**Comment by dpranke** [comment]
Ah, got it. Yes, that looks like a bug.

Side note: I'm curious, are you tripping across this specific example, or is this a simplified version of something? Because in this case, I'd imagine you could just work around this by deleting the backslash?

**Comment by sloanyyc** [comment]
I am using scrapy spider some web page . It contains js with json5 format. Got a lot of string like "http:\\/\\/abc\\/". I don't sure json5 format deal with '\\/'. With json5 parse is more comfortable.

**Comment by dpranke** [comment]
Thanks for the bug report. This should be fixed now on master, v0.6-dev, and in v0.6.2, newly pushed to PyPI.

This also newly handles "\0" correctly, which it didn't before and which I noticed while digging into this.
